public/api/conf/2.0/application.raml

```diff
@@ -0,0 +1,114 @@
+#%RAML 1.0
+---
+
+title: Customs Inventory Linking Exports
+version: 1.0
+protocols: [ HTTPS ]
+baseUri: https://api.service.hmrc.gov.uk/
+mediaType: application/xml
+
+documentation:
+ - title: Overview
+   content: !include docs/overview.md
+ - title: Versioning
+   content: !include https://developer.service.hmrc.gov.uk/api-documentation/assets/common/docs/versioning.md
+ - title: Errors
+   content: !include https://developer.service.hmrc.gov.uk/api-documentation/assets/common/docs/errors.md
+
+uses:
+  sec: https://developer.service.hmrc.gov.uk/api-documentation/assets/common/modules/securitySchemes.raml
+  annotations: https://developer.service.hmrc.gov.uk/api-documentation/assets/common/modules/annotations.raml
+  types: https://developer.service.hmrc.gov.uk/api-documentation/assets/common/modules/types.raml
+
+/customs/inventory-linking/exports:
+  /:
+    post:
+      displayName: Inventory Exports Request
+      description: Submit an Inventory Exports Request
+      headers:
+        Accept:
+          description: Specifies the version of the API that you want to call. See [versioning](/api-documentation/docs/reference-guide#versioning).
+          type: string
+          required: true
+          example: application/vnd.hmrc.1.0+xml
```

Comment:
example should read `application/vnd.hmrc.2.0+xml`, not `application/vnd.hmrc.1.0+xml` (as this is the core of how versioning works)

Reply:
Fixed